toast-tk-runtime/src/main/java/io/toast/tk/runtime/utils/EncryptHelper.java

```diff
@@ -0,0 +1,32 @@
+package io.toast.tk.runtime.utils;
+
+public class EncryptHelper {
+
+	//private static final Logger LOG = LogManager.getLogger(EncryptHelper.class);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)